### ISSUE TITLE
PatchSystem.sh "worked" but didn't add WiFi back

### ISSUE THREAD

**Issue by BenSova** [Ursinia/Mini-Monterey-Patcher#12]
Hi Ben I posted somewhere here that I just tried the pre-802.11ac patcher that just came out about an hour ago.. I booted into the USB and finally typed everything correctly and the patcher finished successfully but when I boot back into Monterey I get no wifi... It don't show up anywhere. I'm using a Mac Pro 5,1 with Supported PHY Modes:	802.11 a/b/g/n.
Am i doing something wrong?  It said it was patched successfully..
Thanks in advance!

_Originally posted by @GitHub3590 in https://github.com/Ursinia/Mini-Monterey-Patcher/issues/11#issuecomment-877556022_

**Comment by BenSova** [comment]
Hey could you run this command in Terminal (not from Recovery, straight from Monterey) and tell me whether or not there's output?

```
cat /System/Library/Extensions/IO80211FamilyLegacy.kext/Contents/PlugIns/AirPortBrcmNIC.kext/Contents/Info.plist | grep pci14e4,4331
```

**Comment by GitHub3590** [comment]
ok Ben doing it now...

**Comment by GitHub3590** [comment]
ok Ben I got this..

Last login: Fri Jul  9 16:06:49 on console
lou@Monterey ~ % cat /System/Library/Extensions/IO80211FamilyLegacy.kext/Contents/PlugIns/AirPortBrcmNIC.kext/Contents/Info.plist | grep pci14e4,4331
				<string>pci14e4,4331</string>
lou@Monterey ~ %

**Comment by Monkiey** [comment]
> ok Ben I got this..
> 
> Last login: Fri Jul 9 16:06:49 on console
> lou@Monterey ~ % cat /System/Library/Extensions/IO80211FamilyLegacy.kext/Contents/PlugIns/AirPortBrcmNIC.kext/Contents/Info.plist | grep pci14e4,4331
> pci14e4,4331
> lou@Monterey ~ %

yeah the issues might me because you are using a MacPro mac pros arent supported by mini Monterey officially you should have used Big Mac patcher: https://github.com/StarPlayrX/bigmac

**Comment by GitHub3590** [comment]
thanks but bigmac2 doesn't work with the wifi original card I have. "802.11 a/b/g/n."

**Comment by GitHub3590** [comment]
when I ran the command Ben gave me it came back..

"<string>pci14e4,4331</string>"

**Comment by GitHub3590** [comment]
I used bigmac for HDMI Audio and Patched Sur for WIFI when I use Big Sur and its works great but I really love keeping up with the latest OS.

**Comment by BenSova** [comment]
Hmm so that means it patched successfully, but it doesn't work for your system. 
Can you run this command in Terminal for me?

```
ioreg -r -n ARPT | grep IOName
```

**Comment by GitHub3590** [comment]
I just got your request Ben, I'm doing it now.
Thanks!

**Comment by GitHub3590** [comment]
Here is the output.

Last login: Sat Jul 10 05:49:31 on console
lou@Monterey ~ % ioreg -r -n ARPT | grep IOName
      "IOName" = "pci14e4,432b"
lou@Monterey ~ %

**Comment by BenSova** [comment]
Ah that's the problem, I'll send you a fix in a little bit.

**Comment by GitHub3590** [comment]
Oh Man Awesome!!!! Thanks Ben!!!

**Comment by GitHub3590** [comment]
Thanks Ben!
    On Saturday, July 10, 2021, 12:52:50 PM EDT, Ben Sova ***@***.***> wrote:  
 
 


Ah that's the problem, I'll send you a fix in a little bit.

**Comment by GitHub3590** [comment]
waiting patiantly for a wifi fix from Ben. 😞

**Comment by Jef-Ferry** [comment]
![FB_IMG_1626030761817](https://user-images.githubusercontent.com/87280319/125207475-df8a7e80-e262-11eb-9f0b-05962d9cba91.jpg)
Congrats!!!

**Comment by BenSova** [comment]
That's for only certain Macs, I'm typing the instructions for @GitHub3590 as we speak.

**Comment by BenSova** [comment]
1. [Download this](https://www.dropbox.com/s/rkaobi2vhu0tien/Mini-Monterey-Patcher-Test.zip?dl=1)
2. Re-patch your USB with it. (You don't need to remake it, you can just patch it again without changing anything)
3. Reboot into the USB, and then open Terminal
4. Type in `/Volumes/"Image Volume"/PatchSystem.sh --wifi=native-plus /Volumes/"Your Volume Name"` (keeping all quotes) then press enter.
5. Once it finishes, reboot and in theory it should work.

**Comment by GitHub3590** [comment]
Wow!!! Doing it now Ben! I will report back! Thanks!!!

**Comment by Jef-Ferry** [comment]
i believe i tried all versions but this one finally worked and now i know i can really update if i need to, thank you so much for your work.
Success!

**Comment by GitHub3590** [comment]
I'm having boot issues.. Nothing new with a 2010 5,1 w/Monterey beta 2 but after 2 successful patches I can't get any machine to boot the 2 drives I'm testing on.. I will keep trying.
Just an update.

**Comment by BenSova** [comment]
I'm pretty sure that was one of the Macs that didn't boot Big Sur without assistance. 
I don't currently support anything pre-2012 with this patcher.

**Comment by GitHub3590** [comment]
I have 3 2012 5,1's and 1 2010 5,1. All mac pro's and Your Patched Sur works great for patching wifi and it works everytime with 11.2.3.
I thought I would let you know.

**Comment by GitHub3590** [comment]
But now i'm sure this fix is locking the boot up every time but that ok Ben, This is all testing and I don't mind it.
I'm wondering is your Patched Sur works on 11.2.3 do you think you can get it to work on Monterey beta 2?
All Mac Pro's here.....

**Comment by Monkiey** [comment]
> I'm pretty sure that was one of the Macs that didn't boot Big Sur without some extra assistance.
> I don't currently support anything pre-2012 with this patcher.

yeah, I would recommend OCLP or Big Mac patcher for that model @GitHub3590

**Comment by GitHub3590** [comment]
Thanks Moniey but the wifi don't work on OCLP or bigmac2 in Monterey with my wifi. "IOName" = "pci14e4,432b"

**Comment by BenSova** [comment]
> yeah, I would recommend OCLP patcher for that model @GitHub3590

OCLP doesn't support Monterey on pre-2012 Macs yet

**Comment by BenSova** [comment]
How did the test WiFi patch go?

**Comment by Monkiey** [comment]
> > yeah, I would recommend OCLP patcher for that model @GitHub3590
> 
> OCLP doesn't support Monterey on pre-2012 Macs yet

oh okay, thanks for the insight

**Comment by GitHub3590** [comment]
Oops I did a typo... Patched-Sur works PERFECT on my Mac Pro 5,1 2012 using Big Sur 11.2.3 Every time!!!!!

**Comment by GitHub3590** [comment]
Well I appreciate the time you put in Ben... I guess I'll have to stay in 11.2.3 for now but I can't stress enough that your Patched-Sur patcher works perfect to get my wifi going on all four of my Mac Pro's.. I only hope there will be a fix that works with Monterey as well.
The patch finishes successfully but I can't boot th OS to see if it works.. lol

**Comment by BenSova** [comment]
Huh, fun, I'll look into it soon.

**Comment by GitHub3590** [comment]
ok Ben thanks!

**Comment by GitHub3590** [comment]
Hey Ben, As I said above your fix wifi patch completes successfully but then I can't boot into the OS anymore... I'm sure it's a simple fix for you and then we can see if your wifi "IOName" = "pci14e4,432b" works on a 5,1 mac pro.
OMG you would make thousands of peeps happy!!

**Comment by GitHub3590** [comment]
I decided to take the drive that won't boot Monterey after the patch Fix and boot it from my 2010 MacBook Pro and it booted fine and the Ethernet works but no wifi as with my Mac Pro 5,1...
No Ethernet on the Mac Pro... Just noting.

**Comment by GitHub3590** [comment]
Just a note:
Monterey beta 3 w/OCLP got my Mac Pro 5,1 Ethernet ports working..
 Now I just need the wifi Ben.
"IOName" = "pci14e4,432b"
(hint hint)

**Comment by Kief5555** [comment]
(I am Kief#2583 On discord.) I have the same mac year, should I install?

**Comment by GitHub3590** [comment]
If you have the same wifi and BT as I have this patcher won't work.
If you have a 2013 or later I think it's supported by Ben's Patcher but I'm not sure.
Cheers

**Comment by BenSova** [comment]
It's not by year, but rather by something about the specific model. For example, my 2012 MBP works fine with it, but there seems to be an issue on your 2012 Mac Pro.

**Comment by GitHub3590** [comment]
True!
Thanks Ben!
